src/libs/KeyboardShortcut/bindHandlerToKeydownEvent/index.native.ts

```diff
@@ -0,0 +1,28 @@
+import getKeyEventModifiers from '../getKeyEventModifiers';
+import BindHandlerToKeydownEvent from './types';
+
+/**
+ * Checks if an event for that key is configured and if so, runs it.
+ */
+const bindHandlerToKeydownEvent: BindHandlerToKeydownEvent = (getDisplayName, eventHandlers, keyCommandEvent, event) => {
```

Comment:
please add return type

Reply:
Return type is defined in `BindHandlerToKeydownEvent` type, so no need to define it twice + the type is void so either way it's too simple to write according to the guidelines 😄